integreat_cms/api/tests/api_tests_utils.py

```diff
@@ -0,0 +1,36 @@
+"""
+This modules contains helper functions for automatically generating view tests.
+"""
+import json
+import re
+
+
+def generate_test_functions(class_variables, endpoint_data):
+    """
+    This function generates test functions for a list of views.
+
+    :param class_variables: The class variable dict in which the functions should be inserted
+    :type class_variables: dict
+
+    :param endpoint_data: The list of endpoints for which the test functions should be generated
+    :type endpoint_data: list
+    """
+    for data in endpoint_data:
+        endpoint, wp_endpoint, fixture = data
+        endpoint_str = re.sub("[^a-zA-Z0-9]", "_", endpoint.strip("/"))
+
+        def test_function(
+            self, endpoint=endpoint, fixture=fixture, wp_endpoint=wp_endpoint
```

Comment:
```suggestion
            self, endpoint=endpoint, wp_endpoint=wp_endpoint, expected_result=expected_result
```
I'd rename `fixture` to `expected_result` to point out that these json files are not intended to be imported as Django fixtures.